### DISCUSSION TITLE
Bad Request: can't parse entities: Character

### DISCUSSION THREAD

**Discussion by trura1985** [alfiosalanitri/server-guardian#11] [General]
hi there!
i got this error "{"ok":false,"error_code":400,"description":"Bad Request: can't parse entities: Character '.' is reserved and must be escaped with the preceding '\\'"}+ exit 1" message when trying use this script, maybe you have some solutions to resolve it please?

**Comment by alfiosalanitri** [comment]
Hi, can you write here the log file content that report last error?

**Reply by trura1985** [reply]
it happens because something like this https://stackoverflow.com/questions/40626896/telegram-does-not-escape-some-markdown-characters
the solution for me was change method parsing from MarkdownV2 to HTML
and another thing that I find - if there are more than 1 simultaneous event (high CPU load and high RAM consumption e.t.c) - only one event is notified.

**Reply by trura1985** [reply]
or maybe I do something wrong? when I want to use all alerts?

**Reply by alfiosalanitri** [reply]
https://github.com/alfiosalanitri/server-guardian/issues/2#issuecomment-1119639902
